pack/urbp.json

```diff
@@ -48,7 +48,7 @@
         "position": 3,
         "quantity": 3,
         "side_code": "runner",
-        "text": "+1[link]\nWhenever you install a non-<strong>AI</strong> <strong>icebreaker</strong>, that <strong>icebreaker</strong> has +2 strength until the end of the turn.",
+        "text": "+1[link]\nWhenever you install a non-<strong>AI</strongnisei.net/> <strong>icebreaker</strong>, that <strong>icebreaker</strong> has +2 strength for the remainder of the turn.",
```

Comment:
:eyes:

Reply:
I'm making new HTML tags, Noah. It's part of my branding efforts.